dev/create-release/releaseutils.py

```diff
@@ -185,6 +185,7 @@ def get_commits(tag):
     "graphx": "GraphX",
     "input/output": CORE_COMPONENT,
     "java api": "Java API",
+    "kubernetes": "Kubernetes",
```

Comment:
this is for the PR title [foo] - I think [k8s] is more widely used, maybe both

Reply:
Can we supply a list/tuple there? I've updated it to `K8S`, but sometimes folks have written `k8s` or `kubernetes` in the PR titles by the looks of it.

Reply:
so looks like this is for both commit title and JIRA component field... which isn't quite perfect (for example, not R here)

but in any case, multiple left value can map to the same right value, it looks like

Reply:
Ah, okay, I misread that previously - updated the mapping, and it looks like the script turns things to lower case anyway, so `k8s` and `kubernetes` ought to cover everything. Thanks!

Reply:
yes, I think this looks right